indico/modules/users/schemas.py

```diff
@@ -16,8 +16,8 @@ class UserSchema(mm.SQLAlchemyAutoSchema):
 
     class Meta:
         model = User
-        fields = ('id', 'identifier', 'first_name', 'last_name', 'email', 'affiliation', 'avatar_bg_color', 'full_name',
-                  'phone')
+        fields = ('id', 'identifier', 'first_name', 'last_name', 'email', 'affiliation', 'full_name',
+                  'phone', 'picture_url')
```

Comment:
how ugly would it be to change it to `avatar_url` everywhere?

Reply:
depends, do we want to change stuff like `picture_metadata` to as well?

Reply:
I wouldn't especially since those are DB columns... but the `picture_url` is just a property IIRC